app/src/main/java/org/wikipedia/feed/news/NewsCardView.java

```diff
@@ -38,8 +38,15 @@ public NewsCardView(@NonNull Context context) {
         super(context);
         inflate(getContext(), R.layout.view_card_news, this);
         ButterKnife.bind(this);
-        newsRecyclerView.addItemDecoration(new RecyclerViewIndicatorDotDecor(DimenUtil.roundedDpToPx(4),
-                DimenUtil.roundedDpToPx(8), DimenUtil.roundedDpToPx(20),
+        setUpIndicatorDots();
+    }
+
+    private void setUpIndicatorDots() {
```

Comment:
It will still have a `checkstyle` warning if you don't put `@SuppressWarnings("checkstyle:magicnumber")` above this method.